### PR TITLE
Feature/sync service

### PR DESCRIPTION
**Pull request by pavax**

### CODE REVIEW

File: sync-service/src/main/resources/application-local.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+stayhome:
```
Comment (mageo) [inline sync-service/src/main/resources/application-local.yaml:1]: I think this should use environment variables and not be defined in a static file.

File: sync-service/src/main/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+feign:
+  client:
+    config:
+      default:
+        connectTimeout: 5000
+        readTimeout: 5000
+        loggerLevel: basic
+
+stayhome:
```
Comment (mageo) [inline sync-service/src/main/resources/application.yaml:17]: I think this should use environment variables and not be defined in a static file.

File: sync-service/src/test/resources/application-test.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+stayhome:
```
Comment (mageo) [inline sync-service/src/test/resources/application-test.yaml:1]: I think this should use environment variables and not be defined in a static file.

**Review by mageo — APPROVED** [review]

File: sync-service/src/main/resources/application-local.yaml (hunk 1 above)
Comment (pavax) [reply sync-service/src/main/resources/application-local.yaml:1]: in generell using spring-boot: every property can always be overrriden by an environment-variable.

**Pull request merged**